### PR TITLE
Stuff the AI will like (hopefully not broken this time)

### PR DESCRIPTION
**Pull request by grinkgo**
## About The Pull Request

Improvements to AI stuff on the ship. Like an extra holopad. And better doors and shutters. And cameras in the ERT shuttle bays.

## Why It's Good For The Game

AI has more control

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by grinkgo** [comment]
> You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

No, _you_ currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/game/objects/machinery/doors/airlock_types.dm [hunk 1]
```diff
@@ -261,6 +261,18 @@
 	icon = 'icons/obj/doors/mainship/securedoor.dmi'
 	req_access = list(ACCESS_MARINE_BRIDGE)
 
+/obj/machinery/door/airlock/mainship/ai
+	name = "\improper AI Core"
+	icon = 'icons/obj/doors/mainship/personaldoor.dmi'
+	req_access = list(ACCESS_MARINE_BRIG, ACCESS_MARINE_LOGISTICS)
```
Comment (DominikPanic) [inline code/game/objects/machinery/doors/airlock_types.dm]: The previous one had req_one_access of 19, which is BRIDGE access, this one requires both brig and logistics.
if that's what you want to do it's fine

Comment (grinkgo) [reply code/game/objects/machinery/doors/airlock_types.dm]: Tbh idk what logistics is used for

Comment (DominikPanic) [reply code/game/objects/machinery/doors/airlock_types.dm]: You should look it up where it's used (job datums have an access list), if you're not sure then you should preserve behavior, as this is changing it.

Comment (grinkgo) [reply code/game/objects/machinery/doors/airlock_types.dm]: ok dad

Comment (Rohesie) [inline code/game/objects/machinery/doors/airlock_types.dm]:
```suggestion
	req_access= list(ACCESS_MARINE_BRIDGE)
```

Comment (grinkgo) [reply code/game/objects/machinery/doors/airlock_types.dm]: ope I cannot spell

**Pull request merged**